File: fetch-scraper/scraper/models/job.py
# pylint: disable-all
# pylint: disable-all
# pylint: disable-all
# pylint: disable-all
"""
Job struct for JobProducer

Non-RPC struct
"""

from dataclasses import dataclass


@dataclass
class JobItem:
    uuid: str
    title: str
    company: str
    url: str
    source_id: str
    location: str = ""
    date_posted: str = ""

File: rpc-cache/models/job.go
package models

type JobItem struct {
	Uuid       string `json:"uuid"`
	Title      string `json:"title"`
	Company    string `json:"company"`
	URL        string `json:"url"`
	SourceID   string `json:"source_id"`
	Location   string `json:"location"`
	DatePosted string `json:"date_posted"`
}
